packages/platform-android/src/commands/runAndroid/__tests__/runOnAllDevices.test.js

```diff
@@ -22,67 +22,49 @@ describe('--appFolder', () => {
   });
 
   it('uses installDebug as default if no arguments', () => {
```

Comment:
Probably title of the test should be updated, there are actually arguments provided now